src/components/MedCentersPage/MedCentersList/MedCenter/CenterServices.tsx/ServicesStyle.tsx

```diff
@@ -60,4 +60,12 @@ export const ServiceItem = styled.div<IServiceProps>`
     `
     margin: 0;
   `}
+
+  @media (max-width: ${STYLE_CONSTANTS.SCREEN.laptop}) {
+    font-size: ${STYLE_CONSTANTS.FONT_SIZE.veryLittle};
```

Comment:
`veryLittle`? 😅 
what about extraSmall?

Reply:
tiny: '10px',
    veryLittle: '12px',
    little: '14px',
    small: '16px',
    medium: '18px',
    large: '20px',
    veryLarge: '24px',
    mobileLarge: '30px'

all size)

Reply:
I would suggest to reduce to 3 main sizes `small`, `medium`, `large`

Reply:
https://github.com/KLIM-nk01/virtual-ambulance/pull/29/commits/2185e1031ed143b4da0204e9661376ef44def020#diff-2442105b2633800dbf77e7ad00fb404ad02847f8d617e6d163eda7ee53fd5dd9  I left 4 basic sizes